homeassistant/components/unifi_direct/device_tracker.py

```diff
@@ -11,12 +16,12 @@
     CONF_HOST, CONF_PASSWORD, CONF_USERNAME,
     CONF_PORT)
 
-REQUIREMENTS = ['pexpect==4.6.0']
+REQUIREMENTS = ['paramiko']
```

Comment:
The release needs to be pinned.

Reply:
Also, we transitioned to put requirements into `manifest.json`. So this line should be removed completely.